manifests/install.yaml

```diff
@@ -82,13 +83,33 @@ data:
       Sync operation details are available at: {{.context.argocdUrl}}/applications/{{.app.metadata.name}}?operation=true .
     name: app-sync-failed
     slack:
-      attachments: "[{\n  \"title\": \"{{ .app.metadata.name}}\",\n  \"title_link\":\"{{.context.argocdUrl}}/applications/{{.app.metadata.name}}\",\n
```

Comment:
The `manifests/install.yaml` file is autogenerate by `make manifests`. 
The source YAML files are located in `builtin/templates`. Please fix error in source yamls and rerun `make templates`

Reply:
Thank you Alex! Done.